words.html

```diff
@@ -64,234 +64,237 @@
 <!-- - - - - - - - - - - - - - - - - -->
 
 <p id="title">
-<span style="font-size:0.75em">THE</span><br>
-EVOLUTION<br>
-<span style="font-size:0.75em">OF TRUST</span>
+EWOLUCJA<br>
+ZAUFANIA
 </p>
 <p id="subtitle">
 
-playing time: 30 min &#8226; by nicky case, july 2017
+czas gry: 30 minut &#8226; autor: nicky case, lipiec 2017
 
 <!-- TRANSLATION NOTE: UNCOMMENT THE FOLLOWING & ADD YOUR OWN NAME! -->
-<!--
+
 <br>
-translated by YOUR NAME HERE
+tłumaczenie: mateusz sienkan, agata grdal
 &#8226;
-<a href="http://ncase.me/trust/">original English version</a>
--->
+<a href="http://ncase.me/trust/">original english version</a>
+
 
 </p>
 
 <p id="loading">
-loading...
+ładowanie...
 </p>
 <p id="loading_done">
-PLAY &rarr;
+GRAJ &rarr;
 </p>
 
 <!-- - - - - - - - - - - - - - - - - -->
 <!-- - - - - - INTRO! - - - - - - - - -->
 <!-- - - - - - - - - - - - - - - - - -->
 
 <p id="intro">
-During World War I, peace broke out.
+Podczas Pierwszej Wojny Światowej,<br>
+przez chwilę zapanował pokój.
 <br><br>
 
-It was Christmas 1914 on the Western Front.<br>
-Despite strict orders <i>not</i> to chillax with the enemy, British<br>
-and German soldiers left their trenches, crossed No Man's Land,<br>
-and gathered to bury their dead, exchange gifts, and play games.
+Było to w czasie świąt 1914 roku, na zachodnim froncie.<br>
+Pomimo całkowitego <i>zakazu</i> bratania się z wrogiem, brytyjscy<br>
+i niemieccy żołnierze opuścili swoje okopy, przekroczyli ziemię niczyją,<br>
+i zebrali się, by pogrzebać zmarłych, wymienić prezentami i pograć w gry.
 <br><br>
 
-Meanwhile: it's 2017, the West has been at peace for decades, and<br>
-wow, we <i>suck</i> at trust. Surveys show that, over the past forty years,
-fewer and fewer people say they trust each other. So here's our puzzle:
+Tymczasem: jest 2017 rok, pokój na zachodzie trwa już od dziesiątek lat,<br>
+i łał, okazuje się, że <i>nie potrafimy</i> sobie ufać. Badania pokazują, że przez ostatnie 40 lat,
+coraz mniej ludzi deklaruje, że ufa innym. Oto zagadka:
 <br><br>
```

Comment:
OK dla mnie spoko, jeśli wychodzimy za słownik to chociaż świadomie :) ja też piszę łał na fejsbukach